.github/workflows/auto-assign.yml

```diff
@@ -0,0 +1,29 @@
+name: Auto-assign maintainer
+
+on:
+  issue_comment:
+    types:
+      - created
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  assign:
+    # Only run on PR comments, not issue comments
+    if: github.event.issue.pull_request
```

Comment:
This can be improved. For example, we don't need to run this workflow if the author association is not a maintainer. To minimize the usage, this workflow should be triggered only when necessary.